config/matrix-config-sample.js

```diff
@@ -1,21 +1,31 @@
-var baseUri = '/ermrest/catalog/1/attributegroup/M:=isa:dataset/isa:dataset_species/vocab:species/name=Mus%20musculus/$M/isa:dataset_anatomy/Y:=vocab:anatomy/$M/isa:dataset_stage/X:=vocab:stage/sort_key::geq::0/sort_key::lt::10000/$M/isa:dataset_experiment_type/Z:=vocab:experiment_type'
+var baseUri =
+  '/ermrest/catalog/1/attributegroup/M:=isa:dataset/isa:dataset_species/vocab:species/name=Mus%20musculus/$M/isa:dataset_anatomy/Y:=vocab:anatomy/$M/isa:dataset_stage/X:=vocab:stage/sort_key::geq::0/sort_key::lt::10000/$M/isa:dataset_experiment_type/Z:=vocab:experiment_type';
 
 var matrixConfigs = {
   '*': {
     xURL: baseUri + '/sort_key:=X:sort_key,id:=X:id,title:=X:name@sort(sort_key)',
     yURL: baseUri + '/id:=Y:id,title:=Y:name@sort(title)',
     zURL: baseUri + '/id:=Z:id,title:=Z:name@sort(title)',
     xysURL: baseUri + '/xid:=X:id,yid:=Y:id;zid:=array(Z:name)',
-    xSource: [{'inbound':['isa','dataset_stage_dataset_id_fkey']},{'outbound':['isa','dataset_stage_stage_fkey']},'id'],
-    xFacetColumn: 'id', // id or title
-    ySource: [{'inbound':['isa','dataset_anatomy_dataset_id_fkey']},{'outbound':['isa','dataset_anatomy_anatomy_fkey']},'id'],
-    yFacetColumn: 'id', // id or title
-    zSource: [{'inbound':['isa','dataset_experiment_type_dataset_id_fkey']},{'outbound':['isa','dataset_experiment_type_experiment_type_fkey']},'id'],
-    zFacetColumn: 'id', // id or title
+    xFacetColumn: 'id',
+    yFacetColumn: 'id',
+    zFacetColumn: 'id',
     catalogId: '1',
     schemaName: 'isa',
     tableName: 'dataset',
     prularDataset: 'datasets',
-    matrixContainer: '.mousematrix-container'
-  }
+    matrixContainer: '.mousematrix-container',
+    styles: {
+      title: 'Mouse Data Summary',
```

Comment:
- Are you using `matrixContainer` and `prularDataset`? If not, please remove them.

- I don't think title and subtitle in "styles" makes sense. Let's just have them in the first level and use `title_markdown` and `subtitle_markdown`.